impeller/entity/shaders/blending/framebuffer_blend.vert

```diff
@@ -5,6 +5,9 @@
 #include <impeller/conversions.glsl>
 #include <impeller/types.glsl>
 
+// Warning: if any of the constant values or layouts are changed in this
+// file, then the hard-coded constant value in
+// impeller/renderer/backend/vulkan/binding_helpers_vk.cc
```

Comment:
Thinking out aloud on how to address this issue (in a separate patch later of course): We should be able to import a single header file from both a C++ TU and a glsl header. The same macro expansion capabilities as the C pre-processor are available in impellerc as well. As the `IMPELLER_DEVICE` macro is set in ImpellerC so we should be able to set it. So we can use that to ifdef away C++/GLSL-isms from the other TU.

Reply:
I believe the default impellerc shaders GN rule set also add the header include paths during the impellerc invocation.

Reply:
Just a nit; But I am just nervous about "please keep these in sync manually" type comments.

Reply:
I think that won't work in this case, because its not that we're setting to binding 64 or w/e, that is just the first available binding in the fragment stage. So adding a new binding to the vertex stage might also change that? Not sure.

What I would really like is to have separate GLLS, Vulkan, and Metal headers. As this would also solve the problem of conditionally using float16.